9.2/Dockerfile.rhel7

```diff
@@ -46,6 +43,9 @@ RUN yum install -y yum-utils gettext && \
     test "$(id postgres)" = "uid=26(postgres) gid=26(postgres) groups=26(postgres)" && \
     chmod -R go+rwx /var/lib/pgsql && chmod go+w /var/run/postgresql
 
+COPY run-*.sh /usr/local/bin/
+COPY contrib /var/lib/pgsql/
```

Comment:
Agreed, we cant move copying the files to the /var/lib/.../ after we chmod the dir.

I think we can probably close this.